docs/modeling-your-data/running-data-models-via-snowplow-bdp/dbt/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Using dbt"
+title: "Custom models"
```

Comment:
Could you expand the title to something like “Running custom dbt models via Snowplow BDP” (for SEO)? And then set `sidebar_label` to “Custom models”